davinci-grid.md

```diff
@@ -30,24 +30,38 @@ to some special commands provided by `ganga`.
 To create your first `ganga` job type the following:
 
 ```python
-j = Job(application=DaVinci(version='v41r2'))
+j = Job(name='First ganga job')
+myApp = prepareGaudiExec('DaVinci','v41r2', myPath='.')
+j.application = myApp
```

Comment:
Just a daft question but is there some motivation to having the myApp object around after constructing the job?
I keep seeing it in Ganga tutorials but I don't know why people always construct the object then assign it rather than doing this in 1 line.
(If this is historical ignore me :P)